File: app/src/main/java/example/activity/ReadAssetsActivity.java
package example.activity;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.widget.TextView;

import com.style.base.BaseActivity;
import com.style.base.BaseActivityPresenter;
import com.style.framework.R;
import com.style.framework.databinding.ActivityUserAgreeBinding;
import com.style.utils.AssetsUtil;
import com.style.utils.StreamUtil;

import java.io.IOException;
import java.io.InputStream;

public class ReadAssetsActivity extends BaseActivity {

    ActivityUserAgreeBinding bd;
    private TextView tv_agree;

    @Override
    public int getLayoutResId() {
        return R.layout.activity_user_agree;
    }

    @Override
    protected BaseActivityPresenter getPresenter() {
        return null;
    }

    @Override
    public void initData() {
        bd = getBinding();

        setToolbarTitle("用户协议");

        tv_agree = (TextView) findViewById(R.id.tv_useragree);
        readData();
    }

    public void readData() {
        try {

            tv_agree.setText(AssetsUtil.getAssetsText(this, "useragree.txt"));
        } catch (IOException e) {

        }
    }
}

File: app/src/main/java/example/activity/ReadAssetsActivity.kt
package example.activity;

import android.os.Bundle
import com.style.base.BaseTitleBarActivity
import com.style.framework.R
import com.style.framework.databinding.ActivityUserAgreeBinding
import com.style.utils.AssetsUtil
import java.io.IOException

class ReadAssetsActivity : BaseTitleBarActivity() {

    private lateinit var bd: ActivityUserAgreeBinding

    override fun onCreate(arg0: Bundle?) {
        super.onCreate(arg0)
        bd = ActivityUserAgreeBinding.inflate(layoutInflater)
        setContentView(bd.root)
        setTitleBarTitle("用户协议")
        readData()
    }

    private fun readData() {
        try {
            bd.tvUseragree.text = AssetsUtil.getAssetsText(this, "useragree.txt")
        } catch (e: IOException) {
            e.printStackTrace()
        }
    }
}
